terraform/modules/ecr/main.tf

```diff
@@ -10,7 +10,7 @@ terraform {
 }
 
 #tfsec:ignore:aws-ecr-repository-customer-key:encryption using KMS CMK not required
-resource "aws_ecr_repository" "this" {
+resource "aws_ecr_repository" "core" {

```

Comment:
I would maybe have called this "app" but no biggie.

Reply:
Ah yes, in this case the `name` property of the resource was `core` so I thought it would be clearer if I matched that 😀

Reply:
I probably would have given the resource the name `app` as well! But again, just personal preference and not worth deleting the ECR over.